codec/testdata/test_tx.go

```diff
@@ -6,6 +6,17 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// NewTestFeeAmount is a test fee amount.
+func NewTestFeeAmount() sdk.Coins {
+	return sdk.NewCoins(sdk.NewInt64Coin("atom", 150))
+}
+
+// NewTestGasLimit is a test fee gas limit.
+func NewTestGasLimit() uint64 {
+	return 100000
+}
+
+// NewTestMsg creates a message for testing with the given signers.
 func NewTestMsg(addrs ...sdk.AccAddress) *TestMsg {
```

Comment:
NewTestMsg is used a bit everywhere in the codebase's tests, not only in `codec/`. I'm inclined to move the whole `testdata/` folder into `<root>/testutil/testdata`

Reply:
go for it!